eslint-plugin/docs/rules/no-missing-print-css.md

```diff
@@ -0,0 +1,34 @@
+# Warn when no print CSS is provided (`@ecocode/no-missing-print-css`)
+
+⚠️ This rule _warns_ in the ✅ `recommended` config.
+
+<!-- end auto-generated rule header -->
+
+## Rule Details
+
+This rule aims to remind developers to use CSS print to limit the number of pages printed therefore indirectly reducing the impact of the web page.
+
+## Examples
+
+Examples of **non-compliant** code for this rule:
+
+```js
+
+<head>
+...
+<link rel="stylesheet" type="text/css" href="styles.css">
+
+...
+</head>
+```
+
+Examples of **compliant** code for this rule:
+
+```js
+<head>
+...
+<link rel="stylesheet" type="text/css" href="screen.css" media="screen">
+<link rel="stylesheet" href="impression.css" type="text/css" media="print">
+...
+</head>
```

Comment:
I don't really know if it's a common practice to define `<head>` tags in JSX. Usually these properties are found in the index.html file even in React, right?